Makefile

```diff
@@ -23,7 +23,11 @@ CHECK_EXAMPLE_TIMESTAMPS = $(patsubst examples/%.html,build/timestamps/check-%-t
 
 TASKS_JS := $(shell find tasks -name '*.js')
 
-CLOSURE_LIB = $(shell node -e 'process.stdout.write(require("closure-util").getLibraryPath())')
+ifeq (CYGWIN,$(findstring CYGWIN,$(OS)))
+  CLOSURE_LIB := $(shell cygpath -u $(shell node -e 'process.stdout.write(require("closure-util").getLibraryPath())'))
```

Comment:
Those tweaks will have to wait till tonight if I am to make them. I did
mean  to use = not :=.
Bill
On 13 May 2015 08:03, "Éric Lemoine" <notifications@github.com> wrote:

> In Makefile
> <https://github.com/openlayers/ol3/pull/3692#discussion_r30206526>:
>
> > @@ -23,7 +23,11 @@ CHECK_EXAMPLE_TIMESTAMPS = $(patsubst examples/%.html,build/timestamps/check-%-t
> >
> >  TASKS_JS := $(shell find tasks -name '*.js')
> >
> > -CLOSURE_LIB = $(shell node -e 'process.stdout.write(require("closure-util").getLibraryPath())')
> > +ifeq (CYGWIN,$(findstring CYGWIN,$(OS)))
> > +  CLOSURE_LIB := $(shell cygpath -u $(shell node -e 'process.stdout.write(require("closure-util").getLibraryPath())'))
>
> I am not sure about the use of :=, as it will only work if the
> closure-util is already installed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openlayers/ol3/pull/3692/files#r30206526>.
>